associacao-brasileira-de-normas-tecnicas-ufrgs.csl

```diff
@@ -1,12 +1,12 @@
 <?xml version="1.0" encoding="utf-8"?>
-<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" initialize="false" demote-non-dropping-particle="never" default-locale="pt-BR">
+<style class="in-text" version="1.0" initialize="false" demote-non-dropping-particle="never" default-locale="pt-BR" xmlns="http://purl.org/net/xbiblio/csl">
   <info>
-    <title>Universidade Federal do Rio Grande do Sul - SBUFRGS - ABNT (autoria completa) (Portuguese - Brazil)</title>
-    <title-short>SBUFRGS-ABNT</title-short>
-    <id>http://www.zotero.org/styles/associacao-brasileira-de-normas-tecnicas-ufrgs</id>
-    <link href="http://www.zotero.org/styles/associacao-brasileira-de-normas-tecnicas-ufrgs" rel="self"/>
-    <link href="http://www.zotero.org/styles/associacao-brasileira-de-normas-tecnicas-ipea" rel="template"/>
-    <link href="https://www.ufrgs.br/bibliotecas/estilo-abnt-sbufrgs-para-gerenciadores-de-referencias/" rel="documentation"/>
+    <title>Universidade Federal do Rio Grande do Sul - SBUFRGS - ABNT 6023:2018 (autoria completa)(Portuguese-Brasil) </title>
+    <title-short>ABNT-UFRGS/EBSCO</title-short>
+    <id>http://www.zotero.org/styles/universidade-federal-do-rio-grande-do-sul-sbufrgs-abnt-6023-2018-autoria-completa</id>
```

Comment:
any particular reason for these changes or do they just come from the editor?

Reply:
Done. Thanks!